src/i18n/fi/common.json

```diff
@@ -128,43 +128,49 @@
     "landingPage": {
       "applyForArea": "Hae aluetta",
       "areaSearches": {
-        "blurb": "Voit vuokrata vapaana olevia maa- ja vesialueita monenlaista käyttöä varten, esim. mastot, maatalousmaat, lisäpihat, hiekkasiilot, keräyslaatikot, liikuntapaikat.",
+        "blurb": "Voit vuokrata vapaana olevia maa- ja vesialueita monenlaista käyttöä varten.",
         "heading": "Alueen vuokraus",
         "label": "Vuokraa alue"
       },
       "heading": "Hae vuokra-aluetta",
       "introduction": "Voit vuokrata Helsingin kaupungilta alueita monenlaiseen käyttöön. Valitse tarkoitukseesi sopivin vaihtoehto.",
       "otherUses": {
         "commercialUses": {
-          "blurb": "A-standit, siltamainospaikat, ulkomainoslaitteet, vaalimainonta",
+          "blurb": "Voit vuokrata kaupungin yleistä aluetta lyhytkestoiseen myyntitoimintaan, vuokrata liiketilasi edustaa esittely- ja myyntitoimintaa varten, tehdä sopimuksen Ketterän kioskin eli liikkuvan elintarvikekioskin toimintaa varten. Voit vuokrata kaupungin siltamainospaikkoja lyhytaikaiseen mainontaan ja tiedottamiseen.",
           "label": "Myynti- ja mainospaikat",
-          "link": "https://www.hel.fi/helsinki/fi/asuminen-ja-ymparisto/tontit/luvat/mainonta/"
+          "link": "https://www.hel.fi/fi/yritykset-ja-tyo/yritykset-ja-yrittajat/yritysten-luvat-ja-tilat"
         },
         "heading": "Muu alueenkäyttö",
         "misc": {
-          "blurb": "Esim. kaupunkiviljely, koirakoulutuskentät, mehiläistarhaus, muu maankäyttö- ja vuokraus, metsästysoikeudet , päiväkotien ulkoilualueet, vesialueen vuokraus ",
           "label": "Muut alueen\u00adkäyttö\u00adsopimukset",
-          "link": "https://www.hel.fi/helsinki/fi/asuminen-ja-ymparisto/tontit/luvat/"
+          "dogs": {
+            "blurb": "Koirankoulutuskentiltä voi vuokrata vakioviikkovuoron koirien koulutukseen.",
```

Comment:
These texts seem to be the correct ones, but the ones above are different - is it waiting for translation?